app/controllers/images_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ImagesController < ApplicationController
+  def new; end
+
+  def create
+    @image = Image.new(image_params)
+    if @image.save
+      redirect_to image_path(@image)
```

Comment:
Let's also include a (success) flash message here. Since you are using `redirect_to` instead of `render`, you will need to use `flash` instead of `flash.now` (since `redirect_to` loads a new page while `render` just renders the template on the current page without reloading)